config/environments/production.rb

```diff
@@ -49,4 +49,8 @@
     logger.formatter = config.log_formatter
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
+
+  config.elasticsearch_client = Elasticsearch::Client.new(
```

Comment:
I feel that, rather than being hardcoded for the three different environments, this should be initialized in `config/environment.rb` based on the contents of the `elasticsearch.yml` file. Feel free to change the format of the `config/elasticsearch.yml` file as needed to make that work; we can adjust our recipes.